runtime/criusupport/criusupport.cpp

```diff
@@ -342,6 +342,9 @@ Java_org_eclipse_openj9_criu_CRIUSupport_checkpointJVMImpl(JNIEnv *env,
 		UDATA success = 0;
 		bool safePoint = J9_ARE_ANY_BITS_SET(vm->extendedRuntimeFlags, J9_EXTENDED_RUNTIME_OSR_SAFE_POINT);
 		bool retryPermitted = vm->checkpointState.maxRetryForNotCheckpointSafe > 0;
+		BOOLEAN syslogFlagNone = TRUE;
```

Comment:
Why `BOOLEAN` and not `bool` like the preceding variables (this is C++ code)?

Reply:
Both types are used in this file/function (there is a `BOOLEAN` in earlier variable declaration, they are local variables, I am fine with either way, it might not worth a new PR though.